plots.py

```diff
@@ -0,0 +1,118 @@
+        )
+    write_html_to_file('most_watched_bugs', most_watched_bugs)
+
+
+def suggestion_in_status_pie():
+    """Creating a pie chart in png format with data on the percentage suggestion statuses"""
+    suggestion_in_status = pd.DataFrame(rd.count_issues_in_status('Suggestion'), columns =['Status', 'Number of issues'])
+    suggestion_in_status['Percentage'] = round(
+        suggestion_in_status['Number of issues'] / suggestion_in_status['Number of issues'].sum() * 100, 2
+        )
+    suggestion_in_status.loc[suggestion_in_status['Percentage'] < 1, 'Status'] = 'Others'
+    suggestion_in_status = suggestion_in_status.groupby('Status').sum()
+    pie_diagram = suggestion_in_status.plot(
+        y='Number of issues',
+        kind="pie",
+        autopct='%1.1f%%',
+        title="Statuses of Suggestions",
+        figsize=(10, 10)
+        )
+    pie_diagram.figure.savefig(f'{PATH}/suggestion_in_status_pie')
+
+
+def bug_in_status_pie():
+    """Creating a pie chart in png format with data on the percentage bugs statuses"""
+    bug_in_status = pd.DataFrame(rd.count_issues_in_status('Bug'), columns =['Status', 'Number of issues'])
+    bug_in_status['Percentage'] = round(
+        bug_in_status['Number of issues'] / bug_in_status['Number of issues'].sum() * 100, 2
+        )
+    bug_in_status.loc[bug_in_status['Percentage'] < 1, 'Status'] = 'Others'
+    bug_in_status = bug_in_status.groupby('Status').sum()
+    pie_diagram = bug_in_status.plot(
+        y='Number of issues',
+        kind="pie",
+        autopct='%1.1f%%',
+        title="Statuses of Bugs",
+        figsize=(10, 10)
+        )
+    pie_diagram.figure.savefig(f'{PATH}/bug_in_status_pie')
+
+
+def completed_issue_destribution_bar():
+    """Creating a bar chart in png format with data on the issues distribution by versions"""
+    completed_issue_destribution = pd.DataFrame(
+        rd.distribution_of_issues_by_fixed_versions(), 
+        columns =['Version', 'Number of issues']
+        )
+    completed_issue_destribution = completed_issue_destribution.loc[completed_issue_destribution['Version'].str.match(r'^\d+[\d\.]*$')]
+    completed_issue_destribution['Version'] = completed_issue_destribution['Version'].str.extract(r'^(\d{1,2}\.+\d{0,2})')
+    completed_issue_destribution = completed_issue_destribution.groupby('Version').sum()
+    bar_diagram = completed_issue_destribution.plot(
+        y='Number of issues',
+        kind='bar',
+        figsize=(10, 10),
+        color=(completed_issue_destribution['Number of issues'] > 100).map({True: 'g',False: 'r'}),
+        legend=False
+        )
+    plt.title("Distribution of completed issues", fontsize=25)
+    plt.xlabel('Version', fontsize=20)
+    plt.ylabel('Number of Issues', fontsize=20)
+    bar_diagram.figure.savefig(f'{PATH}/completed_issue_destribution')
+
+
+if __name__ == "__main__":
+    os.makedirs(PATH,exist_ok=True)
+    most_voted_suggestions_table()
+    most_voted_bugs_table()
```

Comment:
functions_to_call = [most_voted_suggestions_table, most_voted_bugs_table, ...]
for func in functions_to_call:
     func()

Reply:
Done